Exercise_D.js

```diff
@@ -14,7 +14,14 @@
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
+  if (number==0 || number<0 || typeof number !== "number"){
+    return "ERROR";
+  } else if (number >= 1000000){
+    return number;
+  } else {
+    while(number < 1000000){
+      number = number * 10 
+    }
+    return number;
+  }
 }

```

Comment:
Well done this logic is great you could simplify line 17 and line 23
(number<=0 || type...
number *= 10

Well done though the logic works well